### PR TITLE
State that Summary does not support quantiles

### PR DESCRIPTION
**Pull request by arturhoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ with s.time():
   pass
 ```
 
+**Summary support is incomplete**. The python client doesn't store or expose quantile information at this time.
```
Comment (brian-brazil) [inline README.md]: That's a bit strong, this is a complete client library per the spec. I'd remove the first sentence.

**Comment by arturhoo** [comment]
I've probably missed something then. From [the docs](http://prometheus.io/docs/concepts/metric_types/) I was expecting Summary to expose:

* streaming φ-quantiles (0 ≤ φ ≤ 1) of observed events, exposed as `<basename>{quantile="<φ>"}`
* the total sum of all observed values, exposed as `<basename>_sum`
* the count of events that have been observed, exposed as `<basename>_count`

But indeed, the [Guidelines] does mention the client libraries are only encouraged to expose quantiles. From my humble understanding either the docs should be more flexible, or the libraries very strict. @brian-brazil what do you think?

**Comment by brian-brazil** [comment]
The docs need to be relaxed, using summaries without quantiles should be much more common than using them with (quantiles are quite expensive).

**Comment by arturhoo** [comment]
Nice, do you recommend opening a PR on the docs then? I've removed the incomplete part and renamed this PR.

File: README.md [hunk 2]
```diff
@@ -142,6 +142,8 @@ with s.time():
   pass
 ```
 
+The python client doesn't store or expose quantile information at this time.
```
Comment (brian-brazil) [inline README.md]: We're capitalising Python elsewhere, can you do the same for consistency?

**Comment by arturhoo** [comment]
Done.

File: README.md [hunk 3]
```diff
@@ -142,6 +142,8 @@ with s.time():
   pass
 ```
 
+The Python Client doesn't store or expose quantile information at this time.
```
Comment (brian-brazil) [inline README.md]: We don't capitalise client :(

Comment (arturhoo) [inline README.md]: k

**Pull request merged**